rero_ils/modules/documents/dojson/contrib/marc21tojson/loc/model.py

```diff
@@ -353,39 +344,38 @@ def marc21_to_series(self, key, value):
                                                 with_dot=True).rstrip()
                 }]
                 if value.get('v'):
-                    parts = []
-                    for subfield_v in utils.force_list(value.get('v')):
-                        parts.append(remove_punctuation(subfield_v))
+                    parts = [
+                        remove_punctuation(subfield_v) for subfield_v
+                        in utils.force_list(value.get('v'))
+                    ]
+
                     subseriesStatement['subseriesEnumeration'] = [{
                         'value': '/'.join(parts)
                     }]
                 series['subseriesStatement'] = [subseriesStatement]
-            else:
-                if value.get('n'):
-                    if value.get('v'):
-                        string_build = build_string_from_subfields(
-                            value, 'nv')
-                        series['seriesEnumeration'] = [{
-                            'value': remove_punctuation(string_build,
-                                                        with_dot=True).rstrip()
-                        }]
-                    else:
-                        if value.get('n'):
-                            subseriesStatement['subseriesTitle'] = [{
-                                'value': ''.join(
-                                    utils.force_list(value.get('n')))
-                            }]
-                            series['subseriesStatement'] = [subseriesStatement]
-                elif value.get('v'):
-                    parts = []
-                    for subfield_v in utils.force_list(value.get('v')):
-                        parts.append(remove_punctuation(subfield_v))
-
+            elif value.get('n'):
+                if value.get('v'):
+                    string_build = build_string_from_subfields(
+                        value, 'nv')
                     series['seriesEnumeration'] = [{
-                        'value': '/'.join(parts)
+                        'value': remove_punctuation(string_build,
+                                                    with_dot=True).rstrip()
                     }]
-                # marc21.extract_series_statement_from_440_field(value, self)
-
+                elif value.get('n'):
```

Comment:
not need `elif valid.get('n')` because you are already into a parent exact same condition statement. You could replace with `else`